cob_hardware_config/cob4-3/config/base_controller.yaml

```diff
@@ -1,5 +1,5 @@
 ## joint_names
-joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint]
+joint_names: [fl_caster_rotation_joint, fl_caster_r_wheel_joint, b_caster_rotation_joint, b_caster_r_wheel_joint, fr_caster_rotation_joint, fr_caster_r_wheel_joint]
```

Comment:
how did `cob4-3`, `cob4-4`, `cob4-6` work before with the `joint_names` set wrong?